back/app/serializers/web_api/v1/permission_serializer.rb

```diff
@@ -5,4 +5,6 @@ class WebApi::V1::PermissionSerializer < WebApi::V1::BaseSerializer
 
   belongs_to :permission_scope, polymorphic: true
   has_many :groups
+  has_many :permissions_custom_fields
+  has_many :custom_fields
```

Comment:
No quite understanding why both are needed in the serializer

Reply:
I wanted to also include the custom fields, so that the frontend could get e.g. the titles to show the fields of the permission. The permission_custom_fields are also necessary to know the required values. However, I think maybe we can do a nested include instead like `permissions_custom_fields: [:custom_field]`. I'll give that a try.